api/justfile

```
@@ -121,6 +121,11 @@ doc-test: wait-up
       --fail \
       'http://localhost:50280/v1/schema/' > openapi.json
 
+# Run the API schema tests using schemathesis
+test-schema: wait-up
+    just doc-test
+    python -m pip install schemathesis && st run --checks response_schema_conformance openapi.json --base-url http://localhost:50280 && st run --checks response_headers_conformance openapi.json --base-url http://localhost:50280 && st run --checks not_a_server_error openapi.json --base-url http://localhost:50280
```

Lets move the installation of the package to Pipfiles and add some new lines here for legibility.

```suggestion
    st run --checks response_schema_conformance openapi.json --base-url http://localhost:50280 && \
    st run --checks response_headers_conformance openapi.json --base-url http://localhost:50280 && \
    st run --checks not_a_server_error openapi.json --base-url http://localhost:50280
```